sql/scan.go

```diff
@@ -673,15 +674,22 @@ func (n *scanNode) getQVal(col ColumnDescriptor) parser.Expr {
 
 type qnameVisitor struct {
 	*scanNode
-	err error
+	stack []parser.Expr
```

Comment:
We push expressions when `pre==true` and pop expressions when `pre==false`. Now that I think of it, I can avoid the stack by simply looking for `parser.FuncExpr` and then examining its arguments. I'll make that change.